pkg/asset/installconfig/platform.go

```diff
@@ -288,13 +289,24 @@ func (a *platform) libvirtPlatform() (*types.LibvirtPlatform, error) {
 		return nil, err
 	}
 
+	// TODO: Ideally, this would live inside of a closure which is passed to
+	//       asset.GenerateUserProvidedAsset and only called if the environment
+	//       variable isn't present. As this exists, it ruins the abstraction.
```

Comment:
This sort of problem goes away with #556, where all dependencies and defaults are only rendered as needed 🎉